eclass-writing/text.xml

```diff
@@ -659,6 +659,122 @@ src_compile() {
 }
 </codesample>
 
+<p>
+<!-- TODO
+Eclasses may inherit other eclasses to make use of their functionality, and,
+historically there have been instances of eclasses calling
+<c>EXPORT_FUNCTIONS</c> and then inheriting another eclass. 
+the time, and 
+-->
```

Comment:
i would like to put in a sentence or two to outline just _briefly_ that the interaction between `inherit` and `EXPORT_FUNCTIONS` had been not-fully-defined at some point and an implementation-defined behaviour had been dependend on (as I understand https://bugs.gentoo.org/399039). The conclusion of the paragraph is then to quote the PMS ("must not rely on any particular behaviour when inheriting another eclass after a call to EXPORT_FUNCTIONS"), maybe with a note to say: "hey, just put EXPORT_FUNCTIONS at the end from now on, and everybody's happy".